deploy/charts/rook-ceph-cluster/values.yaml

```diff
@@ -361,17 +361,17 @@ cephClusterSpec:
     # deviceFilter:
     # config:
     #   crushRoot: "custom-root" # specify a non-default root label for the CRUSH map
-    #   metadataDevice: "md0" # specify a non-rotational storage so ceph-volume will use it as block db device of bluestore.
+    #   metadataDevice: "nvme0n1" # specify a non-rotational storage so ceph-volume will use it as block db device of bluestore.
     #   databaseSizeMB: "1024" # uncomment if the disks are smaller than 100 GB
     #   osdsPerDevice: "1" # this value can be overridden at the node or device level
     #   encryptedDevice: "true" # the default value for this option is "false"
+    # nodes:
```

Comment:
The comments for a section are usually above the section, but if this is really more clear for nodes we can change it.

Reply:
IMO, it's better to keep it as is.

Reply:
I'm fine to drop it. I just think it would make life easier for other people.  I just took this [line](https://github.com/rook/rook/blob/master/deploy/charts/rook-ceph-cluster/values.yaml#L251) as an example
New to helm charts, and fell with a co-worker over the settings here (It's not just me being dense :sweat_smile: ) . Now where I know, I can say yea sure! that makes sense, but it was a bit painful.  

From the description it is not clear that you have to set `useAllDevices` to false as well  when you configure devices in `nodes:` (Something I found out after the pull request). I will add another commit for this. 

Thanks for reviewing!